test/src/components/Button-test.js

```diff
@@ -26,4 +26,19 @@ describe('Button', () => {
     React.addons.TestUtils.Simulate.click(node);
     customHandleClick.called.should.be.true;
   });
+  it('should take in a color attribute', () => {
+    let coloredButton = render(<Button color={'teal'}>Submit</Button>).findTag('button');
+    coloredButton.props.className.should.contain('teal');
+    coloredButton.getDOMNode().getAttribute('class').should.contain('teal');
+  });
+  it('should take in boolean attributes', () => {
+    let disabledButton = render(<Button disabled>Submit</Button>).findTag('button');
+    disabledButton.props.className.should.contain('disabled');
+    disabledButton.getDOMNode().getAttribute('class').should.contain('disabled');
+  });
+  it('should ignore unknown attributes', () => {
+    let unknownAttrButton = render(<Button fakeValue>Submit</Button>).findTag('button');
+    unknownAttrButton.props.className.should.not.contain('fakeValue');
+    unknownAttrButton.getDOMNode().getAttribute('class').should.not.contain('fakeValue');
+  });
```

Comment:
This test is checking that unknownvalues aren't being added to the button's classes